procfs-core/Cargo.toml

```diff
@@ -18,7 +18,7 @@ serde1 = ["serde"]
 
 [dependencies]
 backtrace = { version = "0.3", optional = true }
-bitflags = "1.2"
+bitflags = { version = "2", features = ["serde"] }
```

Comment:
This will unconditionally bring in `serde`, right?

To avoid that, I think we can have:

```toml
[dependencies]
bitflags = { version = "2" }

[features]
serde1 = ["serde", "bitflags/serde"]
```

(untested by me)